pom.xml

```diff
@@ -98,8 +98,8 @@
     <okio.version>2.2.2</okio.version>
     <retrofit.version>2.5.0</retrofit.version>
     <!-- other properties -->
+    <java.version.release>8</java.version.release>
```

Comment:
Perhaps i am missing some thing, if the `java.version.release` is only used on `Java 11` maven profile. It's value shouldn't be `11` instead of `8` and related to documentation the  `release` tag of the maven plugin is availble since java 9.
thx

Reply:
The element has the right value because we want to build and publish a client compatible with java 8 and next.

Indeed the `release` tag is available since java 9, that's why we have two profiles which allows to switch the plugin settings depending on the underlying JDK.